### PR TITLE
Auth validation

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

**Review by Husain01 — APPROVED** [review]
Code is clean. Functionality works fine. Good to go.

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,43 +1,67 @@
 import React from "react";
 import "./Navbar.css";
 import { useNavigate } from "react-router-dom";
+import { useNotes } from "../../context/notes-context";
```
Comment (Devansu-Yadav) [inline src/components/Navbar/Navbar.jsx:4]: Can add barrel imports to make the imports look a bit cleaner. Check out this article - https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno

**Review by Devansu-Yadav — APPROVED** [review]
Code looks clean. Looks good to merge.

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/components/Navbar/Navbar.jsx:4]: Thanks, I will check this out

**Pull request merged**